### DISCUSSION TITLE
46	FB Überspannung

### DISCUSSION THREAD

**Discussion by musbach** [tbnobody/OpenDTU#2123] [General]
Hallo, ab und zu erhalte ich die Fehlermeldung "Überspannung" für eine Sekunde (oder vielleicht auch etwas länger). Diese Fehlermeldung tritt immer nur morgens auf, so etwa 25 Minuten, nachdem der Wechselrichter gestartet ist. Kann dieses Flackern daher stammen, dass nur 5 Module angeschlossen sind und damit ein Anschluss offen ist?

![Überspannung](https://github.com/tbnobody/OpenDTU/assets/20500637/19ec1dcc-5d75-45b9-8f4d-dda692ec42fe)

**Comment by ghost** [comment]
Hi, 
ich weiß es auch nicht.

Hast du der Hersteller schon angefragt?

Beim Suchen nach diesem Fehler bin ich auf verschiedene Erklärungen gestoßen:
1. Oft trat dieser Fehler Mittags auf, da kam zu viel Spannung von den Modulen und der Wechselrichter war sehr warm.
2. Gelegentlich liest man auch vom Austausch eines gemeinsamen FIs; wobei anzunehmen ist, das durch das Neugerät Kontaktprobleme beseitigt wurden.
Das 1. würde ich ausschließen, da das bei dir morgens auftritt.
Das 2. könnte ich mir irgendwie so vorstellen: Möglicherweise ist bei deiner 230V Anlage etwas nicht ganz in Ordnung und der Wechselrichter wird seine Leistung nicht los, Wenn der WR dann später mit höherer Spannung mehr Leistung einspeisen will, klappt das wohl besser.
Du kannst ja mal von einem Elektriker die Leitung prüfen lassen (Schleifenwiderstand).

**Comment by musbach** [comment]
Kann man das Syslog irgendwohin übertragen, z. B. auf einen USB-Stick oder per WLAN?

**Reply by ghost** [reply]
Ja,
Du kannst einen PC zum Loggen laufen lassen und alles in eine Datei schreiben.
Die DTU mus nur direkt an den Rechner angeschlossen werden.
[openDTU Doku: Zugriff auf die serielle Konsole](https://www.opendtu.solar/firmware/howto/serial_console/)
